### DISCUSSION TITLE
edges between groups

### DISCUSSION THREAD

**Discussion by rwss2021** [statnet/ergm#366] [Q&A] [answered]
I realized this package could be really useful to my current research, but I have a general question on this.
I noticed that the term nodematch() accounts for # of edges within the same groups; however, if I wanted to see whether edges between different groups are significant in the network or not, which function am I supposed to use? or do I need to write a customized one on this?
Thank you for your time!!!

**Comment by sgoodreau** [comment, marked as answer]
Hello - the term that allows for more generalized modeling of differences in mixing patterns among categories is `nodemix`.

**Reply by rwss2021** [reply]
Thanks a lot!!

**Comment by mbojan** [comment]
As usual there is more than one way to go about it:

- `nodemix` as Steve suggested is the most general of the ready-made terms which seems appropriate
- You can create one or more dyadic covariates, each a matrix with 0s for within-group dyads and 1s for dyads between the groups of interest and use it/them with `dyadcov` or `edgecov` terms. The interpretation of the coefficients depends on what set of such created covariates you include in the model, similarly to treatment contrasts in linear models.
- If you are just interested in the between- vs within- then you can simply look at (opposite of) `nodematch` because if a tie is not within-group it must be between-group
- If you have only two groups and code the vertex attribute identifying these groups to have values 0 or 1 you can use `absdiff` (will result with 0s for within-group dyads and 1s for betwen group dyads)

**Comment by martinamorris** [comment]
I'll also note, in case this was your question, that if you want to test whether there is a statistically significant difference in the presence of within-group ties -- vs. the null hypothesis of random mixing -- that's exactly what the `ergm` function is built to do.  You estimate the coefficient on the term of interest (`nodematch`, or the terms noted above), and examine the p-value of the test statistic, which is shown in the usual summary table of the fit object.
